### ISSUE TITLE
Can you export the helper component files as ES6 moduls?

### ISSUE THREAD

**Issue by BergRens11** [casesandberg/react-color#560]
As of now I have to use:

`var { Saturation } = require('react-color/lib/components/common');`

**Comment by ulrichb** [comment]
Yeah. This would enable webpack tree shaking for this (pretty big) library:

> Tree shaking is a term commonly used in the JavaScript context for dead-code elimination. It relies on the static structure of ES2015 module syntax, i.e. import and export. 

^ [https://webpack.js.org/guides/tree-shaking/](https://webpack.js.org/guides/tree-shaking/)

**Comment by jakobsandberg** [comment]
Looks like this is a dupe of #498

**Comment by casesandberg** [comment]
Closing as is dupe.
